README.md

```diff
@@ -9,6 +9,25 @@ We plan to enable image data and time-series data.
 
 Find the documentation [here](https://automl.github.io/Auto-PyTorch/development)
 
+## Workflow
+
+The rough description of the workflow of Auto-Pytorch is drawn in the following figure.
+
+<img src="figs/apt_workflow.png" width="500">
+
+In the figure, **Data** is provided by user and
+**Portfolio** is a predefined pool of machine learning algorithms, e.g. LightGBM and support vector machine, to solve either
+regression or classification task on the provided dataset.
+Then API starts the following procedures:
+1. **Validate input data**: Process each data type, e.g. encoding categorical data, so that Auto-Pytorch can handled
+2. **Create dataset**: Create a dataset that can be handled in this API including cross validation splits
+3. **Evaluate baselines**: Train each algorithm in the predefined pool with a fixed hyperparameter configuration and dummy model from `sklearn.dummy` that represents the worst possible performance
```

Comment:
you can mention "Baselines is a predefined pool of machine learning algorithms, e.g. LightGBM and support vector machine, to solve either regression or classification task on the provided dataset" here.